redi-connect-front/src/components/organisms/RediHero.tsx

```diff
@@ -18,20 +13,22 @@ const RediHero = () => {
   return (
     <Section className="default-background">
       <Container>
-        <Columns vCentered>
-          <Columns.Column size={5}>
+        <Columns>
+          <Columns.Column size={5} className="redi-hero__content">
```

Comment:
Another thing is, please try not to style funcional elements from bulma. This could lead to unpredictable behaviour...

Reply:
removed